File: Frontend/vis-res-pro-6.0.1/src/app/models/view-models/CompanyExpiredInsurancesViewModel.ts

export class CompanyExpiredInsurancesViewModel {
    insuranceType: string;
    numberExpired: number;
    hide: boolean;

    constructor(InsuranceType: string, NumberExpired: number) {
      this.insuranceType = InsuranceType;
      this.numberExpired = NumberExpired;
    }

    // toggle() {
    //   this.hide = !this.hide;
    // }
}

File: Backend/10 - VisRes.Models/ViewModels/Dashboard/CompanyExpiredInsurancesViewModel.cs
using System.ComponentModel.DataAnnotations;

// TODO: GP, Change Vrs.Models.EF to Vrs.Models.ViewModel but throwing weird error. Sort later
namespace VisRes.Models.EF
{
    public partial class CompanyExpiredInsurancesViewModel
    {
        [Key]
        public string InsuranceType { get; set; }

        public int NumberExpired { get; set; }
    }
}